tests/stress/empty.go

```diff
@@ -0,0 +1 @@
+package rpc
```

Comment:
what is this for?

Reply:
without this the project wont compile. Try removing it to see.

Reply:
if you need a go file within this directory, can this be named `stress.go` and could you paste in the copyright header, maybe add a comment as well explaining why this file cannot be deleted in the file itself.

Reply:
Well, the file is named empty since it is empty. It will be required in few folders since this is a go thing. The name empty is a convention that I see other projects are using.